src/js/controllers/app.js

```diff
@@ -59,6 +64,40 @@ module.controller('AppCtrl', ['$rootScope', '$compile', 'rpId', 'rpNetwork',
     $scope.showAnnouncement = store.get('announcement');
   }
 
+  // Set default directory if it has not already been set
+  $scope.fileInputClick = function(txnName, txData) {
+    fileDialog.openDir(function(evt) {
+      $scope.$apply(function() {
+        $scope.defaultDirectory = evt;
+        $scope.$watch('userBlob', function() {
+          if ($scope.userBlob.data && $scope.userCredentials.username) {
+            $scope.userBlob.set('/defaultDirectory', evt);
+            $scope.defaultDirectory = $scope.userBlob.data.defaultDirectory;
```

Comment:
It looks like `$scope.defaultDirectory` is already set on line 71. Do we need this line?